pkg/storaged/warnings.jsx

```diff
@@ -88,12 +90,22 @@ export function find_warnings(client) {
                 content_size: vdo.physical_size
             });
         }
+
+        if (stratis_bdev && (lvol.Size - Number(stratis_bdev.TotalPhysicalSize) - crypto_overhead) > vgroup.ExtentSize) {
+            enter_warning(path, {
+                warning: "unused-space",
+                volume_size: lvol.Size - crypto_overhead,
+                content_size: Number(stratis_bdev.TotalPhysicalSize)
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230825-082034-fda8c7cb-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)